src/dirs.proj

```diff
@@ -1,5 +1,5 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" InitialTargets="RestoreAllPackages;VerifyBuildTools" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="dir.props" />
+  <Import Project="..\dir.props" />
```

Comment:
For completeness I would still keep the dirs.props file here even if all it does is import the root one. It at least allows us to add more specific stuff in the future without updating any other projects.